scripts/globals/spells/trust/monberaux.lua

```diff
@@ -14,16 +14,29 @@ spellObject.onSpellCast = function(caster, target, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
```

Comment:
I have no doubt that downstream users are going to come in and mess with Monberaux for balance and whatever. Can you collect all the various TODOs in here and from his moves so it's clear what he needs to be completed/verified - if that's not going to happen in this PR